network/nxos/nxos_vrrp.py

```diff
@@ -21,7 +21,7 @@
 ---
 
 module: nxos_vrrp
-version_added: "2.1"
```

Comment:
Please revert this change, the `version_added` field should go under `admin_state`